tidb-cloud/scale-tidb-cluster.md

```diff
@@ -61,22 +61,14 @@ To decrease the number of TiDB nodes, take the following steps:
     > Alternatively, you can also click the name of the cluster that you want to scale on the **Clusters** page and click **...** in the upper-right corner.
 
 3. Click **Modify** in the drop-down menu. The **Modify Cluster** page is displayed.
-4. On the **Modify Cluster** page, decrease the number of TiDB nodes.
+4. On the **Modify Cluster** page, decrease the number of TiDB, TiKV, or TiFlash nodes.
 5. Click **Confirm**.
 
-To decrease the number of TiKV or TiFlash nodes, you need to submit a support ticket. The PingCAP support team will contact you and complete the scaling within the agreed time.
-
 > **Warning:**
 >
 > Decreasing TiKV or TiFlash node number can be risky, which might lead to insufficient storage space, excessive CPU usage, or excessive memory usage on remaining nodes.
```

Comment:
@SunRunAway Should we keep the warning or remove it?

Reply:
it's ok to keep it.